src/rlpx/ecies.ts

```diff
@@ -17,7 +17,7 @@ import {
 
 function ecdhX(publicKey: Buffer, privateKey: Buffer) {
   // return (publicKey * privateKey).x
-  return ecdhUnsafe(publicKey, privateKey, true).slice(1)
```

Comment:
I assume that this breaks the tests. The ecdhUnsafe method seems to be removed from v4. Needs investigation

Reply:
@holgerd77 any idea?

Reply:
Here was some issue on ecdhUnsafe from Vinay before, not sure if this is related: https://github.com/cryptocoinjs/secp256k1-node/issues/138

Reply:
Have posted a [question](https://github.com/cryptocoinjs/secp256k1-node/pull/139#issuecomment-628718319) on this over on the `secp256k1-node` library.

Reply:
See answer over on the repo thread